src/openrct2-ui/windows/LoadSave.cpp

```diff
@@ -298,6 +315,136 @@ namespace OpenRCT2::Ui::Windows
+
+            // Draw frame
+            auto& widget = widgets[WIDX_SCROLL];
+            auto frameStartPos = windowPos + ScreenCoordsXY(width - kPreviewWidth - kPadding - 1, widget.top);
+            auto frameEndPos = frameStartPos + ScreenCoordsXY(kPreviewWidth + 1, kPreviewHeight + 1);
+            GfxFillRectInset(dpi, { frameStartPos, frameEndPos }, colours[1], INSET_RECT_F_60 | INSET_RECT_FLAG_FILL_MID_LIGHT);
+
+            // Draw park name
+            {
+                auto namePos = frameStartPos + ScreenCoordsXY{ kPreviewWidth / 2, -kButtonFaceHeight };
+                auto ft = Formatter();
+                ft.Add<StringId>(STR_STRING);
+                ft.Add<const char*>(_preview.parkName.c_str());
+                DrawTextEllipsised(dpi, namePos, kPreviewWidth, STR_WINDOW_COLOUR_2_STRINGID, ft, { TextAlignment::CENTRE });
+            }
+
+            // Draw image, if available
+            bool foundImage = false;
+            for (auto& image : _preview.images)
+            {
+                if (image.type == PreviewImageType::screenshot)
+                {
+                    auto imagePos = frameStartPos + ScreenCoordsXY(1, 1);
+                    drawPreviewImage(image, dpi, imagePos);
+                    foundImage = true;
+                    break;
+                }
+            }
+
+            // Draw OpenRCT2 logo if no preview was found
+            if (!foundImage)
+            {
+                auto imagePos = frameStartPos + ScreenCoordsXY(1, 1);
+                auto colour = ColourMapA[colours[1].colour].dark;
+                GfxDrawSpriteSolid(dpi, ImageId(SPR_G2_LOGO_MONO_DITHERED), imagePos, colour);
+
+                auto textPos = imagePos + ScreenCoordsXY(kPreviewWidth / 2, kPreviewHeight / 2 - 6);
+                DrawTextBasic(
+                    dpi, textPos, STR_NO_PREVIEW_AVAILABLE, {},
+                    { ColourWithFlags{ COLOUR_WHITE }.withFlag(ColourFlag::withOutline, true), TextAlignment::CENTRE });
+                return;
+            }
+
+            auto summaryCoords = frameStartPos + ScreenCoordsXY(0, kPreviewHeight + kListRowHeight);
+
+            // Date
+            {
+                auto ft = Formatter();
+                ft.Add<StringId>(DateFormatStringFormatIds[Config::Get().general.DateFormat]);
+                ft.Add<StringId>(DateDayNames[_preview.day]);
+                ft.Add<int16_t>(_preview.month);
+                ft.Add<int16_t>(_preview.year + 1);
+                DrawTextBasic(dpi, summaryCoords, STR_SUMMARY_DATE, ft);
+                summaryCoords.y += kListRowHeight;
+            }
+
+            // Park Rating
+            {
+                auto ft = Formatter();
+                ft.Add<money64>(_preview.parkRating);
+                DrawTextBasic(dpi, summaryCoords, STR_SUMMARY_PARK_RATING, ft);
+                summaryCoords.y += kListRowHeight;
+            }
+
+            // Cash
+            {
+                auto ft = Formatter();
```

Comment:
This should be left out for "no money" parks.

Reply:
Ohh, nice catch. I totally hadn't thought of that. That'll require an extra bool in the preview struct.